docs/oneliners.rst

```diff
@@ -0,0 +1,347 @@
+.. -*- mode: rst -*-
+
+Oneliners
+=========
+
+Yellowbrick's quick methods are visualizers in a single line of code!
+
+Yellowbrick is designed to give you as much control as you would like over the plots you create, offering parameters to help you customize everything from color, size, and title to preferred evaluation or correlation measure, optional bestfit lines or histograms, and cross validation techniques. To learn more about how to customize your visualizations using those parameters, check out the :doc:`api/index`.
+
+But... sometimes you just want to build a plot with a single line of code!
```

Comment:
I appreciate this intro - it definitely says "the api is powerful, but feel free to try things out quickly"